Okta-Application/.rpdk-config

```diff
@@ -17,5 +17,16 @@
         "target_schemas": [],
         "useDocker": false,
         "protocolVersion": "2.0.0"
+    },
```

Comment:
This change is not going to be compatible with the way we are automating contract tests in the pipeline.

Reply:
This change was only added so I could run ctv2 tests, if it will not break anything I suggest it is left for future use

Reply:
If it only affects ctv2, we can leave it. We run contract tests in the Alpha account using ssm variables with different names populated by CodeBuild job settings.